tests/Meilisearch.Tests/MeilisearchClientTests.cs

```diff
@@ -126,6 +127,19 @@ public async Task DeleteTasks()
             Assert.Equal($"?uids=1,4&afterStartedAt={formattedDate}", task.Details["originalFilter"].ToString());
         }
 
+        [Fact]
+        public async Task SwapIndexes()
+        {
+            var swaps = new List<IndexSwap> { new IndexSwap("indexA", "indexB") };
+            var response = await _defaultClient.SwapIndexesAsync(swaps);
+            var task = await _defaultClient.WaitForTaskAsync(response.TaskUid);
+
+            response.TaskUid.Should().Be(task.Uid);
+            response.Type.Should().Be(TaskInfoType.IndexSwap);
+            task.Status.Should().Be(TaskInfoStatus.Failed);
```

Comment:
Why make a request that will fail?

Reply:
Because, in this case, I'm not interested in seeing if the indexes' swap will work. It is not up to this SDK to test this.

So, since we don't have a mock structure yet in this SDK, I can easily send a failing request and then assert all the methods I want, like which type of task I receive, if the Details contain the values I want, and so on... It will be almost the same if we call a successful request!